### PR TITLE
Styles and tests

### PR DESCRIPTION
**Pull request by Miguelus373**
### What this PR does:

- Adds the routing system with 3 pages
- Adds styles to all these pages
- Adds tests to all components and methods

### CODE REVIEW

File: src/components/App.test.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
-});
+// test('renders learn react link', () => {
+//   render(<App />);
+//   const linkElement = screen.getByText(/learn react/i);
+//   expect(linkElement).toBeInTheDocument();
+// });
```
Comment (MuriloRoque) [inline src/components/App.test.js]: - [x] Please remove this file, as you are not using it.

File: src/logic/__tests__/operate.test.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  });
+
+  it('Returns "Error" if any divison by zero is attempted', () => {
+    expect(operate('85', '0', '÷')).toBe('Error');
+
+    expect(operate('0', '0', '÷')).toBe('Error');
+
+    expect(operate('-30', '0', '÷')).toBe('Error');
+  });
+});
```
Comment (MuriloRoque) [inline src/logic/__tests__/operate.test.js]: - [x] Add some other scenarios with wrong calculations to make your tests even more solid, like "Doesn't return the wrong result of the sum of two numbers" by adding `5` and `10` not to be `6` for example.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @Miguelus373 :wave: 

### You have done amazing work with this project, you have mastered testing in React, I believe you are ready to move forward. But before that, what do you think if we improve your project a little bit so your tests are even more solid? Please check my comments on your code below :smile: 

Kindly make the requested changes and submit them for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by wathigo — APPROVED** [review]
## APPROVED :heavy_check_mark: 
Great job handling both the negative and positive scenarios in the tests.

**You may move on.**

**Happy Learning :technologist: 
Simon Wathigo**

**Pull request merged**